server/app/backbones.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from typing import List
+from app import deps, schemas, crud
+from app.model import Vector
+
+
+router = APIRouter()
+
+
+@router.get("/vectors/backbones", response_model=List[schemas.Vector])
```

Comment:
Given this is in the "/vectors/" route I think I would prefer you to put this function in the `vectors.py` file.